lib/camping/session.rb

```diff
@@ -1,3 +1,6 @@
+require 'rack/session'
+class InsecureSecret < Exception #:nodoc: all
```

Comment:
```suggestion
class InsecureSecretError < Exception #:nodoc: all
```

Reply:
Better to be explicit in the name (and would need to be changed further down too)

Reply:
Changed.